### DISCUSSION TITLE
Can a map be rendered into an existing view?

### DISCUSSION THREAD

**Discussion by coolbluewater** [maplibre/maplibre-rs#296] [General]
Is it possible to render a map into an existing view (such as a swiftui view as part of an existing swiftui macos/ios app) on all platforms, including apple, windows, linux and android in their respective "normal" app frameworks? And can multiple maps be rendered in separate views?

**Comment by maxammann** [comment]
All of that will be possible in the future.

The foundations are ready, but this is far from being production ready.
